test/drivers/second_quantization/gaussiand/test_driver_gaussian_log.py

```diff
@@ -35,7 +35,7 @@ def test_log_driver(self):
             [
                 "#p B3LYP/6-31g Freq=(Anharm) Int=Ultrafine SCF=VeryTight",
                 "",
-                "CO2 geometry optimization B3LYP/cc-pVTZ",
+                "CO2 geometry optimization B3LYP/6-31g",
```

Comment:
This was an inconsistency in the description. This has no effect on the calculation outcome.